src/ParachainRegistry.sol

```diff
@@ -13,6 +13,9 @@ interface IRegistry {
         uint32 id;
         address owner;
         bytes palletInstance;
+        uint256 weightToFee;
```

Comment:
Please add comments to these fields to explain why they are there and how they are used.